docs/tutorials/search-v3.md

```diff
@@ -0,0 +1,31 @@
+# Amundsen `/v2/search`
+
+The goal of this tutorial is to explain what the new search proxy offers, how to transition into using `v2/search` out of the box as well as how to customize it ahead of deprecation of the old elasticsearch proxy.
+
+## Overview
+
+Amundsen `/v2/search` supports [fuzziness](https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html#fuzziness), word [stemming](https://www.elastic.co/guide/en/elasticsearch/reference/8.1/stemming.html), and [boosted search ranking](https://www.elastic.co/guide/en/elasticsearch/reference/8.1/query-dsl-rank-feature-query.html) based on usage. In order for search to function, metadata for all searchable resources configured is indexed into Elasticsearch with a databuilder task and then Elasticsearch is queried via the search service. In Elasticsearch there are two main concepts that are fundamental to understanding how search works, [mappings](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping.html) and [search queries](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-your-data.html). Mappings define how fields are indexed into Elasticsearch by specifying the types of fields and how text fields are analyzed. The queries should then be written with the mapping in mind since there certain field types are required to perform certain types of queries efficiently.
```

Comment:
The versions are a bit confusing. I think it'd be good to clear up that `/v2/search` is a new unified search API between frontend and search service. And the thing that supports fuzziness etc is the newest proxy.

Reply:
Also see my other comment about doing semver.